comparisons/forms/file_upload/a11y.txt

```diff
@@ -0,0 +1 @@
+Usage: Safe for production.
```

Comment:
Probably want to add a label to the two controls, like "Upload a profile photo" or something like that